packages/vaadin-menu-bar/src/vaadin-menu-bar-interactions-mixin.js

```diff
@@ -56,15 +56,29 @@ export const InteractionsMixin = (superClass) =>
 
     /** @private */
     _themeChanged(theme) {
+      this._buttons.forEach((button) => {
+        const itemTheme = this.__stringOrArrayToString(button.item && button.item.theme);
+        if (itemTheme) {
+          button.setAttribute('theme', itemTheme);
+        } else if (theme) {
+          button.setAttribute('theme', theme);
+        } else {
+          button.removeAttribute('theme');
+        }
+      });
```

Comment:
This logic should be removed from here as we now handle it in the other mixin - no need to duplicate.
So just removing the lines below should be enough (and only set sub menu theme).

Reply:
Fixed, and the menu-bar tests in web-components and flow-components still pass, so looks good.